analytics/report/src/report_store/output.html

```diff
@@ -0,0 +1,83 @@
+<!-- HTML template to render result from report_generator.py -->
+
+<!DOCTYPE html>
+<html lang="en">
```

Comment:
report store is a class where it can take a file and store it. it can mock the storage for now. i.e
report_store.store(report)